src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -734,12 +734,8 @@ public CSS() {
         valueConvertor.put(CSS.Attribute.BACKGROUND_ATTACHMENT,
                            valueMapper);
         Object generic = new CssValue();
-        int n = CSS.Attribute.allAttributes.length;
-        for (int i = 0; i < n; i++) {
-            CSS.Attribute key = CSS.Attribute.allAttributes[i];
-            if (valueConvertor.get(key) == null) {
-                valueConvertor.put(key, generic);
-            }
+        for (Attribute key : Attribute.allAttributes) {
```

Comment:
It looks like the whole method references the Attribute class via CSS prefix.

Reply:
Good catch. Updated